src/pages/multichain.tsx

```diff
@@ -292,43 +287,101 @@ function MultichainPage() {
                   </p>
                   <p className="tw-paragraph md:tw-lead-sm mb-0">
-                    To create ckBTC, a user transfers their bitcoin to a ckBTC
-                    address provided by their wallet (e.g. see functionality
-                    provided at the NNS). Their ckBTC twin can then be sent to
-                    any other ckBTC address, almost instantly for a tiny fee, or
-                    directly to a standard Bitcoin address, causing the bitcoin
-                    twin to return to its native form.
+                    The UX of a Web3 service can provide very fast, and totally
+                    decentralized, edge routing for transactions. For example,
+                    when a canister smart contract has created a transaction for
+                    execution on another blockchain, the UX (e.g JavaScript
+                    running in the web browser) can retrieve the signed
+                    transaction by making a call to the smart contract, and then
+                    push it to the online API of a node in the destination
+                    network, then poll for the result.
+                  </p>
+                </ContentCard>
+
+
+                <ContentCard id="eth-cronjobs">
+                  <h3 className="tw-heading-4 md:tw-heading-3 mb-0">
+                    Use ICP to create crypto cron jobs on Ethereum
+                  </h3>
+                  <p className="tw-paragraph md:tw-lead-sm mb-0">
+                    A key challenge involved when hosting DeFi and other
+                    services using Ethereum smart contracts, is the secure
+                    initiation of regular jobs. Oftentimes, smart contract jobs
+                    are initiated using external scripts running on insecure
+                    clouds such as Amazon Web Services, where the private key
+                    they maintain to make the calls is vulnerable, in the same
+                    way the private keys used by hot wallets are.
+                  </p>
+                  <p className="tw-paragraph md:tw-lead-sm mb-0">
+                    The Internet Computer provides a solution. Canister smart
+                    contracts have an API that allows them to schedule their
+                    secure automatic execution for as long as needed into the
+                    future. Using EVM RPC, canisters can be used to initiate
+                    time-based calls into Ethereum smart contracts too – without
+                    a private key being made vulnerable.
+                  </p>
+
+                  <p className="flex gap-2 flex-wrap">
+                    <Status type="pending">pending</Status>
+                  </p>
+                </ContentCard>
+
+                <ContentCard id="bitcoin-ordinals">
+                  <h3 className="tw-heading-4 md:tw-heading-3 mb-0">
+                    Schnorr-enabled Bitcoin Ordinal inscription, BRC20 tokens,
```

Comment:
```suggestion
                    Bitcoin Ordinal inscription, BRC20 tokens,
```

Reply:
I think mentioning Schnorr here is a technical detail that should not matter at the level of the heading.